README.md

```diff
@@ -2,46 +2,48 @@
 
 The repo contains the pachyderm helm chart.
 
-Status: **Experimental**
-
-## Folder Structure
-
-```
-pachyderm - the helm chart itself
-test - Go based tests for the helm chart
-```
-
 ## Developer Guide
 To run the tests for the helm chart, run the following:
 
-```
-make test
+```shell
+$ make test
 ```
 
 # Diff against `pachctl`
 
 To see how this Helm chart and `pachctl deploy` differ, one can do
 something similar to the following:
 
- 1. Generate a `pachctl` manifest with `pachctl deploy google
-    BUCKET-NAME 10 --dynamic-etcd-nodes 1 -o yaml --dry-run >
-    pachmanifest.yaml`
- 2. Generate a Helm manifest with `helm template -f
-    examples/gcp-values.yaml ./pachyderm > helmmanifest.yaml`
- 3. Visually diff the two.
+1. Generate a `pachctl` manifest with
+
+```shell
+$ pachctl deploy googleBUCKET-NAME 10 --dynamic-etcd-nodes 1 -o yaml --dry-run > pachmanifest.yaml
+```
+
+1. Generate a Helm manifest with
```

Comment:
The number is resetting after each item. Is this perhaps related to indentation or whitespace?

Reply:
This is correct. Numbered lists in markdown should always be listed as "1." and the markdown render will automatically generate the numbers.

Reply:
In case you were more curious on how linters implement this:

https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md#md029---ordered-list-item-prefix

Reply:
Agreed that numbered list numerals should all be 1 in the Markdown source; what I mean is that in GitHub they all _render_ as 1: take a look at https://github.com/pachyderm/helmchart/blob/a8ee4e8ca3df8cc97312f57c140a2eb5f3e73613/README.md

You need to indent the code blocks. For an example, see https://gist.github.com/robert-uhl/e0f0d866c338f8d89e5fb6b957f706da

Reply:
Fixed!